assets/scss/components/_avatar.scss

```diff
@@ -112,25 +112,30 @@ Class                     | Description
 //
 // Type variants
 //
+$iconOffset: 4px;
 .avatar--noPhoto {
-	background-color: $C_textSecondary;
 	position: relative;
 	text-indent: initial;
 
-	&:before {
-		background: $C_textSecondaryInverted;
-		border-radius: 999px;
-		bottom: 0;
-		content: '';
-		display: block;
-		left: 0;
+	.svg {
+		bottom: -#{$iconOffset};
 		position: absolute;
-		top: 0;
+		text-align: center;
 		width: 100%;
 	}
 
-	&.avatar--person {
-		background-position-y: 6px;
+	.svg-icon {
+		fill: $C_textHint;
```

Comment:
I'd use `@color-svg($C_textHint)` here - this will help keep fill/stroke rules in one place if we update how icon styling works